pkg/utils/docker.go

```diff
@@ -106,64 +106,106 @@ const (
 )
 
 // DockerCompose to set up the Codewind environment
-func DockerCompose(tempFilePath string, tag string) {
+func DockerCompose(dockerComposeFile string, tag string) {
+	setupDockerComposeEnvs(tag, "")
+	cmd := exec.Command("docker-compose", "-f", dockerComposeFile, "up", "-d", "--force-recreate")
+	output := new(bytes.Buffer)
+	cmd.Stdout = output
+	cmd.Stderr = output
+	if err := cmd.Start(); err != nil { // after 'Start' the program is continued and script is executing in background
+		DeleteTempFile(dockerComposeFile)
+		errors.CheckErr(err, 101, "Is docker-compose installed?")
+	}
+	fmt.Printf("Please wait whilst containers initialize... %s \n", output.String())
+	cmd.Wait()
```

Comment:
Should we be checking the return code from Wait() to get the exit code for the process? Wait() will return an error if the command failed.
We check for certain known error strings but we don't check the commands exit code.